[main] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron -knative-prow-robot

/cc we don't use Prow
/assign we don't use Prow

Produced by: knative-sandbox/knobots/actions/update-deps

Details:
```
bumping golang.org/x/sync 036812b...886fb93:
  > 886fb93 A+C: delete AUTHORS and CONTRIBUTORS
  > 0de741c x/sync/errgroup: make note of the default errgroup goroutine limit (unlimited)
  > 0976fa6 x/sync/errgroup: add TryGo and SetLimit to control concurrency
bumping k8s.io/client-go 14fa49e...11f68c5:
  > 11f68c5 Update dependencies to v0.23.9 tag
bumping k8s.io/gengo 4627b89...397b4ae:
  > 397b4ae Merge pull request # 227 from SimonBaeumer/add-gitignore
  > 940203f Merge pull request # 226 from SimonBaeumer/add-link-to-code-generator
  > 6697a9b Add IDE config files to .gitignore
  > 6c88d16 Add link to code-generator tree in introduction text
  > 7b3c0fe Add .idea/ to .gitignore
bumping k8s.io/apiextensions-apiserver 7e2da33...cf300a6:
  > cf300a6 Update dependencies to v0.23.9 tag
bumping knative.dev/hack 65c463a...9dabf7c:
  > 9dabf7c Update community files (# 206)
  > 782bbaa drop warning (# 204)
  > 92c7e36 Fix the buoy invocation (# 203)
  > 123a278 :broom: Migrate to using the `go run` instead of `go install` (# 172)
  > a645040 Update community files (# 201)
  > 81e30c2 remove references to git markdown checking (# 199)
  > e3be8da Update community files (# 197)
bumping k8s.io/code-generator 31c00a6...31c00a6:
bumping k8s.io/apimachinery 276a8a7...276a8a7:
bumping k8s.io/klog/v2 43cc75f...0990e81:
  > 0990e81 Merge pull request # 342 from pohly/harshanarayana-reviewer
  > 33351c0 Merge pull request # 341 from pohly/contextual-logging-state-fix
  > 442891b OWNERS: add harshanarayana
  > ea66a13 Merge pull request # 337 from pohly/testing-logger-stop
  > c2d5a45 contextual logging: enable by default again
  > 7a070b4 Merge pull request # 333 from harshanarayana/tests/GIT-275-add-additional-test-for-kv
  > 9405f8e ktesting: stop using testing.T when test completes
  > 265aa25 Merge pull request # 327 from pohly/buffer-logger
  > 280fc18 GIT-275: add tests for int and struct keys
  > 78f38f6 ktesting: fix type assertion
  > ca252b1 Merge pull request # 328 from pohly/fatal-backtrace-revert
  > 325472d ktesting: capture log data in memory
  > 66544b3 ktesting: use black box testing
  > 49e17d5 Merge pull request # 332 from harshanarayana/bug/GIT-331/fix-variable-shadown
  > c5c3132 klog.Fatal: document behavior and alternatives
  > 6b3fb0b Merge pull request # 329 from pohly/apidiff-fix
  > 9c48b7d Merge pull request # 330 from pohly/logtools-removal
  > e4329d2 GIT-331: fix shadowing key from the kv pair
  > c977919 klog.Fatal: revert dumping of all stack backtraces to stderr
  > ea35802 Merge pull request # 320 from pohly/state-restore
  > 49d73a5 GitHub: use apidiff with more recent Go
  > e9884d2 remove hack/tools
  > a585df9 Merge pull request # 326 from pohly/remove-trim-duplicates
  > 3c90bf9 refactor global state
  > 788efcd Merge pull request # 325 from pohly/logr-marshaler
  > cebb190 internal: remove unused TrimDuplicates
  > 296f5e6 save and restore state
  > 42bf7a2 Merge pull request # 322 from pohly/kobjs-performance
  > 369b848 support logr.Marshaler
  > cc2ce06 Merge pull request # 315 from pohly/logcheck-regexp
  > 87ca629 Merge pull request # 324 from pohly/trim-duplicates
  > 8a3e48e replace KObjs with KObjSlice
  > c697110 Merge pull request # 314 from pohly/stop-flush-daemon-clarification
  > 5079582 logcheck: fix detection of invalid * regexp in filter
  > 7a827a0 Merge pull request # 319 from pohly/klog-help-texts
  > 4c3943c use faster MergeKVs
  > ff92e06 test: add benchmark for KObjs
  > d07cbbf Merge pull request # 313 from pohly/logcheck-contextual-logging
  > 47f254f StopFlushDaemon: document flushing on shutdown
  > 2910cb1 README.md: clarify -logtostderr
  > 6f47a32 klogr: simplify test failure message
  > 4a4a786 Merge pull request # 297 from pohly/logcheck-contextual-logging
  > 7179178 hack/tools: drop dependency on golangci-lint
  > e2432e8 serialize: add MergeKVs as faster TrimDuplicates alternative
  > b46e95d Merge pull request # 311 from pohly/contextual-logging-stable
  > 1876dee logcheck: harmonize report messages
  > e6d69a5 serialize: benchmark and test TrimDuplicates
  > 7b1c318 Merge pull request # 310 from pohly/fork-clock
  > 36e44e7 promote non-test APIs to stable
  > 216a02e logcheck: add check for With* and NewContext
  > e3af9af dependencies: avoid k8s.io/utils, fork clock code instead
  > 317fe52 logcheck: optionally warn about using direct klog logging
  > 41995aa logcheck: detect if V().Enabled()
  > 5b4c326 logcheck: warn about functions with both Logger and Context
  > 567c037 logcheck: support logr.Logger
  > c4c35ff logcheck: support import renaming and improve klog.V(2) handling
  > df531eb logcheck: support running as golangci-lint plugin
  > 3f08024 logcheck: also check for format specifier in calls like klog.Info
  > 0c36cf7 logcheck: move into package
  > 93980ca logcheck: add test cases for parameter check
bumping k8s.io/api e08b72c...f89a761:
  > f89a761 Update dependencies to v0.23.9 tag
bumping knative.dev/eventing 0eb566a...5a8f707:
  > 5a8f707 Promote delivery timeout to beta (# 6472)
  > 840fce7 chore: update sugar controller ConfigMap example (# 6460)
  > 7e30288 Ignore trailing slashes for Broker addresses (# 6471)
  > 32dea96 update trace of apiserversource (# 6458)
  > 480c8cb Adds receiver options for configuring read and write timeouts message receiver (# 6306)
  > 37dc195 Port Pingsource test to rekt (# 6463)
  > 15dd7ca 🧹 Remove dead variables & config for presubmit markdown checking (# 6452)
  > e5405de Update community files (# 6462)
  > c8435ed :broom: Accommodate to knative-sandbox/reconciler-test# 294 (# 6451)
  > 5a21cd4 Update community files (# 6453)
  > 365a012 Update community files (# 6449)
bumping knative.dev/serving 3666ce7...bfa381a:
  > bfa381a Update net-gateway-api nightly (# 13180)
  > 9574ddb Update net-contour nightly (# 13179)
  > 875937b upgrade to latest dependencies (# 13177)
  > 782df98 Qp options config additions only (# 13162)
  > 034035b add paul to party, update sched through 2022 (# 13174)
  > a2cf188 Add e2e test for DomainMapping working with websocket. (# 13077)
  > 02d89d8 Update net-contour nightly (# 13171)
  > 77340bb Update net-kourier nightly (# 13170)
  > f846b2c bumping go.mod to v1.18 (# 13169)
  > 8fc6d87 Update net-istio nightly (# 13167)
  > 0b694b0 Update net-certmanager nightly (# 13165)
  > b3fb541 Update net-gateway-api nightly (# 13164)
  > 1f849ab Update net-kourier nightly (# 13166)
  > d0887e7 Update community files (# 13163)
  > bb043fa updating name of activation-scale (# 13161)
  > addd55a Add active-min-scale annotation (# 13136)
  > c82be27 Update net-istio nightly (# 13156)
  > f428628 Update net-gateway-api nightly (# 13155)
  > 63523f9 Enable refactoring queue-proxy binary with out-of-tree extensions (# 13133)
  > 8f351ad Update net-gateway-api nightly (# 13152)
  > 1656d0d Update net-certmanager nightly (# 13151)
  > 08c4062 Update net-kourier nightly (# 13150)
  > 2a22c44 upgrade to latest dependencies (# 13147)
  > 808923d upgrade to latest dependencies (# 13144)
  > ca011f8 Update net-certmanager nightly (# 13142)
  > 50a87e9 Stop using knative.dev/pkg/pool, use errgroup instead (# 13143)
  > 5467217 remove defunct markdown lint and linking variables (# 13137)
  > ed1397d Update `changeset.Get` according to pkg update (# 13141)
  > d1f8e8d bump go version in kind-e2e tests to v1.18 (# 13140)
  > dc552e5 Update community files (# 13138)
  > 38973e5 Update net-gateway-api nightly (# 13134)
  > 044d947 Update net-kourier nightly (# 13132)
  > 017b9d0 Don't included git info in our containers (# 13130)
  > 167199a QP shim main (# 13128)
  > d366049 Update community files (# 13129)
  > 97517d3 Add test flag --ingress-class (# 13117)
  > ed2ddf2 Update net-gateway-api nightly (# 13124)
  > 3ce8ebb Update net-gateway-api nightly (# 13119)
  > 57c2a04 Update net-contour nightly (# 13120)
bumping knative.dev/client bfdc0a2...664b899:
  > 664b899 upgrade to latest dependencies (# 1714)
  > 559a4bd upgrade to latest dependencies (# 1711)
  > 05ccf8e Fix volume creation for PVC (# 1709)
  > 526d680 Update community files (# 1710)
  > ed5fed5 upgrade to latest dependencies (# 1708)
  > 2f1eefe Add check for PVC status in E2E tests (# 1707)
  > 452863f remove defunct markdown linting variable (# 1705)
  > cbf710a Update community files (# 1706)
  > 5251985 Update community files (# 1703)
bumping knative.dev/networking f087178...d2c739d:
  > d2c739d upgrade to latest dependencies (# 698)
  > 20876d7 bump go.mod to v1.18 (# 694)
  > 19a4157 remove defunct markdown linting variable (# 696)
  > 8baa0ac upgrade to latest dependencies (# 692)
  > d6a574e Update community files (# 697)
  > cddb0ff Stop using knative.dev/pkg/pool, use errgroup instead (# 695)
  > 1f4c9ce Bump Go version to 1.18 for CI (# 693)
  > 5ea1a8e Update community files (# 691)
  > 30b5dfe Update community files (# 690)
bumping knative.dev/pkg e60d250...a01dfed:
  > a01dfed bump to k8s 0.23.9 (# 2565)
  > 449f7e6 Update community files (# 2564)
  > ed7ab1d upgrade to latest dependencies (# 2559)
  > 6c9c1c6 remove defunct markdown linting variable (# 2552)
  > 3764d73 fix linting (# 2555)
  > a286510 upgrade to latest dependencies (# 2554)
  > 1a5e168 Update community files (# 2553)
  > c488837 Add reconcile for ValidatingWebhookConfiguration and MutatingWebhookConfiguration. (# 2550)
  > 1ad02a5 Drop error from changeset.Get signature (# 2549)
  > 473ba31 Bump min go version to  1.18 && read build info from embedded binary (# 2548)
  > f1f36a2 Fix subresource update logic. (# 2546)
  > 418e138 Update community files (# 2545)
```

carlisia reviewed (dismissed):
To whomever added:

```
/cc we don't use Prow
/assign we don't use Prow
```

YOU ARE THE BEST

knative-automation committed:
upgrade to latest dependencies

bumping golang.org/x/sync 036812b...886fb93:
  > 886fb93 A+C: delete AUTHORS and CONTRIBUTORS
  > 0de741c x/sync/errgroup: make note of the default errgroup goroutine limit (unlimited)
  > 0976fa6 x/sync/errgroup: add TryGo and SetLimit to control concurrency
bumping k8s.io/client-go 14fa49e...11f68c5:
  > 11f68c5 Update dependencies to v0.23.9 tag
bumping k8s.io/gengo 4627b89...397b4ae:
  > 397b4ae Merge pull request # 227 from SimonBaeumer/add-gitignore
  > 940203f Merge pull request # 226 from SimonBaeumer/add-link-to-code-generator
  > 6697a9b Add IDE config files to .gitignore
  > 6c88d16 Add link to code-generator tree in introduction text
  > 7b3c0fe Add .idea/ to .gitignore
bumping k8s.io/apiextensions-apiserver 7e2da33...cf300a6:
  > cf300a6 Update dependencies to v0.23.9 tag
bumping knative.dev/hack 65c463a...9dabf7c:
  > 9dabf7c Update community files (# 206)
  > 782bbaa drop warning (# 204)
  > 92c7e36 Fix the buoy invocation (# 203)
  > 123a278 :broom: Migrate to using the `go run` instead of `go install` (# 172)
  > a645040 Update community files (# 201)
  > 81e30c2 remove references to git markdown checking (# 199)
  > e3be8da Update community files (# 197)
bumping k8s.io/code-generator 31c00a6...31c00a6:
bumping k8s.io/apimachinery 276a8a7...276a8a7:
bumping k8s.io/klog/v2 43cc75f...0990e81:
  > 0990e81 Merge pull request # 342 from pohly/harshanarayana-reviewer
  > 33351c0 Merge pull request # 341 from pohly/contextual-logging-state-fix
  > 442891b OWNERS: add harshanarayana
  > ea66a13 Merge pull request # 337 from pohly/testing-logger-stop
  > c2d5a45 contextual logging: enable by default again
  > 7a070b4 Merge pull request # 333 from harshanarayana/tests/GIT-275-add-additional-test-for-kv
  > 9405f8e ktesting: stop using testing.T when test completes
  > 265aa25 Merge pull request # 327 from pohly/buffer-logger
  > 280fc18 GIT-275: add tests for int and struct keys
  > 78f38f6 ktesting: fix type assertion
  > ca252b1 Merge pull request # 328 from pohly/fatal-backtrace-revert
  > 325472d ktesting: capture log data in memory
  > 66544b3 ktesting: use black box testing
  > 49e17d5 Merge pull request # 332 from harshanarayana/bug/GIT-331/fix-variable-shadown
  > c5c3132 klog.Fatal: document behavior and alternatives
  > 6b3fb0b Merge pull request # 329 from pohly/apidiff-fix
  > 9c48b7d Merge pull request # 330 from pohly/logtools-removal
  > e4329d2 GIT-331: fix shadowing key from the kv pair
  > c977919 klog.Fatal: revert dumping of all stack backtraces to stderr
  > ea35802 Merge pull request # 320 from pohly/state-restore
  > 49d73a5 GitHub: use apidiff with more recent Go
  > e9884d2 remove hack/tools
  > a585df9 Merge pull request # 326 from pohly/remove-trim-duplicates
  > 3c90bf9 refactor global state
  > 788efcd Merge pull request # 325 from pohly/logr-marshaler
  > cebb190 internal: remove unused TrimDuplicates
  > 296f5e6 save and restore state
  > 42bf7a2 Merge pull request # 322 from pohly/kobjs-performance
  > 369b848 support logr.Marshaler
  > cc2ce06 Merge pull request # 315 from pohly/logcheck-regexp
  > 87ca629 Merge pull request # 324 from pohly/trim-duplicates
  > 8a3e48e replace KObjs with KObjSlice
  > c697110 Merge pull request # 314 from pohly/stop-flush-daemon-clarification
  > 5079582 logcheck: fix detection of invalid * regexp in filter
  > 7a827a0 Merge pull request # 319 from pohly/klog-help-texts
  > 4c3943c use faster MergeKVs
  > ff92e06 test: add benchmark for KObjs
  > d07cbbf Merge pull request # 313 from pohly/logcheck-contextual-logging
  > 47f254f StopFlushDaemon: document flushing on shutdown
  > 2910cb1 README.md: clarify -logtostderr
  > 6f47a32 klogr: simplify test failure message
  > 4a4a786 Merge pull request # 297 from pohly/logcheck-contextual-logging
  > 7179178 hack/tools: drop dependency on golangci-lint
  > e2432e8 serialize: add MergeKVs as faster TrimDuplicates alternative
  > b46e95d Merge pull request # 311 from pohly/contextual-logging-stable
  > 1876dee logcheck: harmonize report messages
  > e6d69a5 serialize: benchmark and test TrimDuplicates
  > 7b1c318 Merge pull request # 310 from pohly/fork-clock
  > 36e44e7 promote non-test APIs to stable
  > 216a02e logcheck: add check for With* and NewContext
  > e3af9af dependencies: avoid k8s.io/utils, fork clock code instead
  > 317fe52 logcheck: optionally warn about using direct klog logging
  > 41995aa logcheck: detect if V().Enabled()
  > 5b4c326 logcheck: warn about functions with both Logger and Context
  > 567c037 logcheck: support logr.Logger
  > c4c35ff logcheck: support import renaming and improve klog.V(2) handling
  > df531eb logcheck: support running as golangci-lint plugin
  > 3f08024 logcheck: also check for format specifier in calls like klog.Info
  > 0c36cf7 logcheck: move into package
  > 93980ca logcheck: add test cases for parameter check
bumping k8s.io/api e08b72c...f89a761:
  > f89a761 Update dependencies to v0.23.9 tag
bumping knative.dev/eventing 0eb566a...5a8f707:
  > 5a8f707 Promote delivery timeout to beta (# 6472)
  > 840fce7 chore: update sugar controller ConfigMap example (# 6460)
  > 7e30288 Ignore trailing slashes for Broker addresses (# 6471)
  > 32dea96 update trace of apiserversource (# 6458)
  > 480c8cb Adds receiver options for configuring read and write timeouts message receiver (# 6306)
  > 37dc195 Port Pingsource test to rekt (# 6463)
  > 15dd7ca 🧹 Remove dead variables & config for presubmit markdown checking (# 6452)
  > e5405de Update community files (# 6462)
  > c8435ed :broom: Accommodate to knative-sandbox/reconciler-test# 294 (# 6451)
  > 5a21cd4 Update community files (# 6453)
  > 365a012 Update community files (# 6449)
bumping knative.dev/serving 3666ce7...bfa381a:
  > bfa381a Update net-gateway-api nightly (# 13180)
  > 9574ddb Update net-contour nightly (# 13179)
  > 875937b upgrade to latest dependencies (# 13177)
  > 782df98 Qp options config additions only (# 13162)
  > 034035b add paul to party, update sched through 2022 (# 13174)
  > a2cf188 Add e2e test for DomainMapping working with websocket. (# 13077)
  > 02d89d8 Update net-contour nightly (# 13171)
  > 77340bb Update net-kourier nightly (# 13170)
  > f846b2c bumping go.mod to v1.18 (# 13169)
  > 8fc6d87 Update net-istio nightly (# 13167)
  > 0b694b0 Update net-certmanager nightly (# 13165)
  > b3fb541 Update net-gateway-api nightly (# 13164)
  > 1f849ab Update net-kourier nightly (# 13166)
  > d0887e7 Update community files (# 13163)
  > bb043fa updating name of activation-scale (# 13161)
  > addd55a Add active-min-scale annotation (# 13136)
  > c82be27 Update net-istio nightly (# 13156)
  > f428628 Update net-gateway-api nightly (# 13155)
  > 63523f9 Enable refactoring queue-proxy binary with out-of-tree extensions (# 13133)
  > 8f351ad Update net-gateway-api nightly (# 13152)
  > 1656d0d Update net-certmanager nightly (# 13151)
  > 08c4062 Update net-kourier nightly (# 13150)
  > 2a22c44 upgrade to latest dependencies (# 13147)
  > 808923d upgrade to latest dependencies (# 13144)
  > ca011f8 Update net-certmanager nightly (# 13142)
  > 50a87e9 Stop using knative.dev/pkg/pool, use errgroup instead (# 13143)
  > 5467217 remove defunct markdown lint and linking variables (# 13137)
  > ed1397d Update `changeset.Get` according to pkg update (# 13141)
  > d1f8e8d bump go version in kind-e2e tests to v1.18 (# 13140)
  > dc552e5 Update community files (# 13138)
  > 38973e5 Update net-gateway-api nightly (# 13134)
  > 044d947 Update net-kourier nightly (# 13132)
  > 017b9d0 Don't included git info in our containers (# 13130)
  > 167199a QP shim main (# 13128)
  > d366049 Update community files (# 13129)
  > 97517d3 Add test flag --ingress-class (# 13117)
  > ed2ddf2 Update net-gateway-api nightly (# 13124)
  > 3ce8ebb Update net-gateway-api nightly (# 13119)
  > 57c2a04 Update net-contour nightly (# 13120)
bumping knative.dev/client bfdc0a2...664b899:
  > 664b899 upgrade to latest dependencies (# 1714)
  > 559a4bd upgrade to latest dependencies (# 1711)
  > 05ccf8e Fix volume creation for PVC (# 1709)
  > 526d680 Update community files (# 1710)
  > ed5fed5 upgrade to latest dependencies (# 1708)
  > 2f1eefe Add check for PVC status in E2E tests (# 1707)
  > 452863f remove defunct markdown linting variable (# 1705)
  > cbf710a Update community files (# 1706)
  > 5251985 Update community files (# 1703)
bumping knative.dev/networking f087178...d2c739d:
  > d2c739d upgrade to latest dependencies (# 698)
  > 20876d7 bump go.mod to v1.18 (# 694)
  > 19a4157 remove defunct markdown linting variable (# 696)
  > 8baa0ac upgrade to latest dependencies (# 692)
  > d6a574e Update community files (# 697)
  > cddb0ff Stop using knative.dev/pkg/pool, use errgroup instead (# 695)
  > 1f4c9ce Bump Go version to 1.18 for CI (# 693)
  > 5ea1a8e Update community files (# 691)
  > 30b5dfe Update community files (# 690)
bumping knative.dev/pkg e60d250...a01dfed:
  > a01dfed bump to k8s 0.23.9 (# 2565)
  > 449f7e6 Update community files (# 2564)
  > ed7ab1d upgrade to latest dependencies (# 2559)
  > 6c9c1c6 remove defunct markdown linting variable (# 2552)
  > 3764d73 fix linting (# 2555)
  > a286510 upgrade to latest dependencies (# 2554)
  > 1a5e168 Update community files (# 2553)
  > c488837 Add reconcile for ValidatingWebhookConfiguration and MutatingWebhookConfiguration. (# 2550)
  > 1ad02a5 Drop error from changeset.Get signature (# 2549)
  > 473ba31 Bump min go version to  1.18 && read build info from embedded binary (# 2548)
  > f1f36a2 Fix subresource update logic. (# 2546)
  > 418e138 Update community files (# 2545)

Signed-off-by: Knative Automation <automation@knative.team>

carlisia commented:
If I close this, would another automated PR to update the dependencies pop up?

gabo1208 commented:
The next deps update PR will pop but with more recen deps
